com.icons8.Lunacy.yml

```diff
@@ -0,0 +1,63 @@
+app-id: com.icons8.Lunacy
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+command: lunacy
+build-options:
+  no-debuginfo: true
+tags:
+  - proprietary
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=cups
+  - --socket=pulseaudio
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-run/gvfs
+  - --filesystem=xdg-run/gvfsd
+  - --filesystem=xdg-config/fontconfig:ro
+  - --talk-name=org.gtk.vfs.*
+  # Network needed for remote asset importing
+  - --share=network
+cleanup:
+  - opt/icons8/lunacy
+  - Lunacy.deb
+  - data.tar.xz
+modules:
+  - name: lunacy
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin
+      - install -D lunacy /app/bin/lunacy
+      - install -D /usr/bin/ar -t /app/bin
+      - mkdir -p /app/lib
+      - ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so /app/lib
+      - install -D apply_extra /app/bin/apply_extra
+
+    sources:
+      - type: extra-data
+        filename: Lunacy.deb
+        url: https://lun-eu-static.s3.eu-central-003.backblazeb2.com/setup/Lunacy.deb
+        sha256: a66dd7cf50c71490b96d3f1d6e2c88da79a24c6810fb7052466a08decf8202d7
+        size: 206367108
+        only-arches:
+          - x86_64
+        x-checker-data:
+          type: rotating-url
+          url: https://lun-eu-static.s3.eu-central-003.backblazeb2.com/setup/Lunacy.deb
+
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - ar x Lunacy.deb
+          - tar xf data.tar.xz --no-same-owner
+          - mv opt/icons8/lunacy/* .
+          - mkdir -p export/share/applications
+          - install -Dm644 Assets/LunacyLogo.png export/share/icons/hicolor/256x256/com.icons8.Lunacy.png
+          - sed 's/Icon=.*/Icon=com.icons8.Lunacy/' usr/share/applications/lunacy.desktop | sed 's/Exec=.*/Exec=\/app\/bin\/lunacy %u/' > export/share/applications/com.icons8.Lunacy.desktop
+          - rm -rf control.tar.xz data.tar.xz debian-binary Lunacy.deb opt
```

Comment:
`Exec` shouldn't be an absolute path.

But you should provide both a .desktop and an icon (svg or 128x128)

Reply:
tbf I'm still not sure how it works with extra data, so let's see what the builder says

Reply:
Ah, the icon is 1024x1024. I tried scalable (which it is not) and 256x256, thinking it'd get picked up and scaled properly.

I'm used to making custom icons in SVG and for specific sizes, although that was long ago at this point in time. :wink: